src/components/IndicatorCard/IndicatorCard.js

```diff
@@ -1,153 +1,42 @@
 import React from 'react';
```

Comment:
Previously there was an `IndicatorCard` that showed either a single number (on the cities page) or two numbers (on the all cities pages).

Now the two cards are sufficiently different that they are two different components. However, we still use `<IndicatorCard>`, and it just selects which card to show.